inc/taxonomy-content-language.php

```diff
@@ -74,6 +74,11 @@ function wmf_get_and_maybe_create_current_language_term(): ?WP_Term {
  * @return \WP_Term|null
  */
 function wmf_create_current_language_term(): ?WP_Term {
+	// When this function is called from the REST API, the admin taxonomy
+	// utilities may not be available. This fix ensures that the content
+	// language functionality works in the block editor.
+	require_once ABSPATH . 'wp-admin/includes/taxonomy.php';
```

Comment:
Should this be gated in an `if ( ! function_exists() )`? I know it'll work either way, but not aware whether there's any theoretical downside to calling this when not needed

Reply:
Makes sense. Intuitively I think of `require_once` as including that same check, but being explicit adds clarity.